<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DeviceMessage</class>
 <widget class="QWidget" name="DeviceMessage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>673</width>
    <height>430</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QPushButton" name="btnSeeDevice">
       <property name="minimumSize">
        <size>
         <width>250</width>
         <height>51</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>250</width>
         <height>51</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>微软雅黑</family>
         <pointsize>10</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>查看设备</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnAddDevice">
       <property name="minimumSize">
        <size>
         <width>250</width>
         <height>51</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>250</width>
         <height>51</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>微软雅黑</family>
         <pointsize>10</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>添加设备</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnDeleteDevice">
       <property name="minimumSize">
        <size>
         <width>250</width>
         <height>51</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>250</width>
         <height>51</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>微软雅黑</family>
         <pointsize>10</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>删除设备</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnGetDevice">
       <property name="minimumSize">
        <size>
         <width>250</width>
         <height>51</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>250</width>
         <height>51</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>微软雅黑</family>
         <pointsize>10</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>查找设备</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
